frontend/src/components/ContentPage/index.js

```diff
@@ -1,13 +1,23 @@
 import React from "react";
+import { useHistory } from "react-router-dom";
 import PropTypes from "prop-types";
 import Form from "../Form";
+import { List, ListItem } from "@code4ro/taskforce-fe-components";
 import {
   Hero,
   Accordion,
   SocialsShare
 } from "@code4ro/taskforce-fe-components";
 
 function ContentPage({ page, subPage }) {
+  const history = useHistory();
+  const navigate = slug => {
+    // Fix SecurityError of pushState on History
+    const cleanPageSlug = `/${page.slug}/${slug}`.replace(/\/+$/, "");
+    const navTarget = page.slug !== cleanPageSlug ? cleanPageSlug : page.slug;
+    history.push(navTarget);
```

Comment:
nit: Isn't this a bit redundant? I think we could always navigate the clean slug
```suggestion
    const cleanPageSlug = `/${page.slug}/${slug}`.replace(/\/+$/, "");
    history.push(cleanPageSlug);
```